DMTR-51.tex

```diff
@@ -1,240 +1,578 @@
+  }
+
 \end{itemize}
 
-\paragraph{[DRP-00-05]}
+The test team agreed that all measurements feel within acceptable tolerances, and therefore regard the test case as having been passed. \jira{DM-13056} and \jira{DM-13057} will be scheduled for further investigation as part of the regular development cycle.
+
+\subsection{DRP-00-25}
+\label{sect:detail-drp-00-25}
+
+The input dataset and analysis tools used for this test case were the same as for DRP-00-15 (see \S\ref{sect:detail-drp-00-15}).
+
+The test team agreed that:
+
 \begin{itemize}
-\item
-\begin{note}
-Data Facility to provide content.
-\end{note}
+
+  \item{PSF models correctly predict the ellipticities of stars over the field of view.}
+  \item{Photometric and astrometric measurements are consistent with reference catalogs.}
+  \item{Forced and unforced measurements are consistent.}
+  \item{The slope of the stellar locus appears appropriate.}
+
 \end{itemize}
 
-\subsubsection{Deviations from test cases/procedures}
-\paragraph{[TEST-CASE-ID]}
-This paragraph shall identify by their unique identification, the test cases in which one or more deviations occured and shall provide:
+However, aperture corrections were not found to be consistent between photometry algorithms.
+This inconsistency was outside acceptable tolerances, and has been ticketed for further investigation as \jira{DM-13058}.
+
+This test case is regarded as ``partially passed'': its overall goal has been accomplished, but understanding the origin of the inconsistent aperture corrections will require further work which will be undertaken during the upcoming developing cycle.
+
+\subsection{DRP-00-30}
+\label{sect:detail-drp-00-30}
+
+The input dataset used for this test case were the same as for DRP-00-15 (see \S\ref{sect:detail-drp-00-15}).
+
+The test was carried out as described.
+All required artefacts were accessible.
+Data was inspected by the Data Release Production Science Lead, who certified that artifacts were correctly matched and that backgrounds were not over-subtracted.
+
+This test case is therefore successfully passed.
+
+\subsection{DRP-00-35}
+\label{sect:detail-drp-00-35}
+
+The input dataset used for this test case were the same as for DRP-00-15 (see \S\ref{sect:detail-drp-00-15}).
+
+The test was carried out as described.
+All required artefacts were accessible.
+Data was inspected by the Data Release Production Science Lead, who certified that artifacts were correctly matched, satellite trails and ghosts were properly rejected, and that backgrounds were not over-subtracted.
+
+It was noted that:
+
 \begin{itemize}
-\item A description of the deviation (e.g. substitution of any resource, procedural step not followed, scchedule deviation, etc)
-\item The rationale of the deviation
-\item An assessment of the deviation impact on the validity of the test case
+
+  \item{We note that the background model is stored separately from the image, but is readily accessible through the same mechanism;}
```

Comment:
It's not obvious to me what "mechanism" is being referred to here.

Reply:
The mechanism used to access the image. :-) But yeah, point taken.